week0/daejun/nginx/nginx.conf

```
@@ -0,0 +1,16 @@
+events { }
+
+  http {
+    upstream board {
```

기능상 문제는 없지만, docker network dns hostname과 겹쳐 혼돈의 여지가 있을 수 있습니다.

해당 내용에 동의 하시면 commit suggestion 버튼을 눌러주시면 됩니다.
```suggestion
    upstream board-api {
```